### PR TITLE
Equality constraints implimented under DQ_QuadprogSolver

### PR DESCRIPTION
**Pull request by qlin960618**
The implimentation of the equality constraints from guidence of @mmmarinho

### CODE REVIEW

File: dqrobotics/solvers/__init__.py [hunk 1]
```diff
@@ -8,10 +8,25 @@
     class DQ_QuadprogSolver(DQ_QuadraticProgrammingSolver):
         def __init__(self):
             DQ_QuadraticProgrammingSolver.__init__(self)
+            self.equality_constraints_tolarence = np.finfo(np.float64).eps
```
Comment (mmmarinho) [inline dqrobotics/solvers/__init__.py]: tolerance

File: dqrobotics/solvers/__init__.py [hunk 2]
```diff
@@ -8,10 +8,25 @@
...
             pass
 
+        def set_equality_constraints_tolarence(self, tolarence):
```
Comment (mmmarinho) [inline dqrobotics/solvers/__init__.py]: Tolerance (and all others)

**Review by mmmarinho — CHANGES_REQUESTED** [review]
Please fix the typos I spotted and maybe other typos you might find.

File: dqrobotics/solvers/__init__.py (hunk 1 above)
Comment (qlin960618) [reply dqrobotics/solvers/__init__.py]: Sorry, spelled it wrong the fist time then i just somehow convinced myself it is spelled that way afterward....

**Comment by qlin960618** [comment]
The type should be fixed now.

**Review by mmmarinho — APPROVED** [review]

**Pull request merged**